src/main/liberty/config/server.xml

```diff
@@ -3,11 +3,8 @@
 
     <!-- Enable features -->
     <featureManager>
-        <feature>concurrent-3.1</feature>
-        <feature>restfulWS-4.0</feature>
-        <feature>cdi-4.1</feature>
-        <feature>websocket-2.2</feature>
-        <feature>jndi-1.0</feature>
+        <feature>concurrent-3.0</feature>
+        <feature>restfulWS-3.1</feature>
     </featureManager>
```

Comment:
Like the dependency removals, these features aren't needed. This also switches from the upcoming Jakarta EE 11 features to the already available EE 10 features. Since the demo doesn't specifically focus on EE 11, it's best to use EE 10 since it's already available.